releases/ampd/2025-04-03-ampd-v1.6.0.md

```diff
@@ -38,13 +38,21 @@ cosmwasm_contract = 'axelar1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq
 type = 'MultisigSigner'
```

Comment:
update version to v1.7.0?

Reply:
Well this was wrong for 1.6.0 as well

Reply:
we'll ask verifiers to update to v1.7.0 directly anyways. we can mention that this applies to both v1.6 and v1.7